start.ps1

```diff
@@ -65,6 +82,20 @@ if ($null -ne $dbs -And $dbs.Length -gt 0)
 
 Write-Host "Started SQL Server."
 
+# run powershell scripts before starting SQL service
```

Comment:
that should be "after starting SQL service", right?

Reply:
Fixed this now 🙂